_posts/2024-07-09-datafusion-40.0.0.md

```diff
@@ -0,0 +1,450 @@
+---
+layout: post
+title: "Apache Arrow DataFusion 40.0.0 Released"
+date: "2024-07-09 00:00:00"
+author: alamb
+categories: [release]
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+<!-- see https://github.com/apache/datafusion/issues/9602 for details -->
+
+## Introduction
+
+We recently [released DataFusion 40.0.0]. This blog highlights some of the many
+major improvements since we [released DataFusion 34.0.0]
+and a preview of where the community is thinking about improving in the next 6 months.
```

Comment:
Not sure we care about the style/voice, but we can make it come across more energetic. Like "We are proud to announce a recent release of DF 40.0.0. The new release brings a lot of improvements and this blog highlights some of the major ones since DF 34 [...]"

(obviously, a matter of preference)

Reply:
Excellent idea -- I tried to strengthen it in 0fb4fd0